pkgs/applications/science/logic/anylogic/default.nix

```diff
@@ -0,0 +1,67 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  gnutar,
+  steam-run,
+  makeDesktopItem,
+  copyDesktopItems,
+}: let
+  desktopItem = makeDesktopItem {
+    desktopName = "AnyLogic Personal Learning Edition";
+    genericName = "AnyLogic Personal Learning Edition - for beginners and students";
+    categories = ["Science"];
+    exec = "anylogic";
+    name = "anylogic-ple";
+    icon = "anylogic-ple";
+  };
+in
+  stdenv.mkDerivation rec {
+    pname = "anylogic-ple";
+    version = "8.8.2";
+
+    src = fetchurl {
+      url = "https://files.anylogic.com/${pname}-${version}.linux.x86_64.tgz.bin";
+      sha256 = "0n372fq6q1pr0y3qxwdf5xxx9phmlyrjjgji861qilrqdg4cmzcp";
+    };
+
+    nativeBuildInputs = [copyDesktopItems gnutar];
+    buildInputs = [steam-run];
+
+    unpackPhase = ":";
+    installPhase = ''
+      runHook preInstall
+      tail -n +374 $src > anylogic_extracted.tgz
+      tar zxf anylogic_extracted.tgz
+      rm anylogic_extracted.tgz
+      cd ./anylogic
+      pwd
+      printf "directory and ready to run.\n\n"
+      chmod -R a+wr plugins/com.anylogic.examples_*
+      cd ..
+      mkdir -p $out/opt
+      mv anylogic $out/opt
```

Comment:
Why not put things into $out/share directly?

Reply:
What do you mean?